File: src/targets/php/http2/fixtures/application-json.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$body = new http\Message\Body;
$body->append(json_encode([
  'number' => 1,
  'string' => 'f"oo',
  'arr' => [
    1,
    2,
    3
  ],
  'nested' => [
    'a' => 'b'
  ],
  'arr_mix' => [
    1,
    'a',
    [
        'arr_mix_nested' => [
                
        ]
    ]
  ],
  'boolean' => false
]));
$request->setRequestUrl('https://httpbin.org/anything');
$request->setRequestMethod('POST');
$request->setBody($body);

$request->setHeaders([
  'content-type' => 'application/json'
]);

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();

File: src/targets/rust/reqwest/fixtures/application-json.rs
use serde_json::json;
use reqwest;

#[tokio::main]
pub async fn main() {
    let url = "https://httpbin.org/anything";

    let payload = json!({
        "number": 1,
        "string": "f\"oo",
        "arr": (1, 2, 3),
        "nested": json!({"a": "b"}),
        "arr_mix": (1, "a", json!({"arr_mix_nested": json!([])})),
        "boolean": false
    });

    let client = reqwest::Client::new();
    let response = client.post(url)
        .header("content-type", "application/json")
        .json(&payload)
        .send()
        .await;

    let results = response.unwrap()
        .json::<serde_json::Value>()
        .await
        .unwrap();

    println!("{}", results);
}
